tests/puppeteer/pages/FO/FObasePage.js

```diff
@@ -116,10 +116,12 @@ module.exports = class Home extends CommonPage {
 
   /**
    * Change currency in FO
-   * @param currency
+   * @param isoCode
+   * @param symbol
    * @return {Promise<void>}
    */
-  async changeCurrency(currency = 'EUR €') {
+  async changeCurrency(isoCode = 'EUR', symbol = '€') {
+    const currency = isoCode === symbol ? isoCode : `${isoCode} ${symbol}`;
     await Promise.all([
```

Comment:
Could you please add a comment to explain why there is this behavior ?

Reply:
done